src/signature.rs

```diff
@@ -66,6 +67,23 @@ pub use rsa::{
     RSA_PKCS1_3072_8192_SHA384_VERIFY,
 };
 
+/// A public key signature.
+pub struct Signature {
+    value: [u8; 64],
+}
+
+impl<'a> Signature {
+    /// Initialize signature from slice of bytes.
+    pub fn new(signature_bytes: [u8; 64]) -> Signature {
```

Comment:
Please mark this as `#[doc(hidden)]` with a comment explaining why it has to be public. (It doesn't make sense to have `new` be part of the public API, but we need to construct `Signature`s from other modules within *ring* so it has to be public until Rust's module system is extended to support what we want.)